ebuild-writing/users-and-groups/text.xml

```diff
@@ -17,8 +17,12 @@ First, check the <uri link="https://api.gentoo.org/uid-gid.txt">UID/GID
 assignment list</uri> and choose a free UID/GID in the range between
 101 and 499. If you are adding a user and a group with the same name,
 use the same number for their UID and GID, respectively. When in
-doubt,
-take the next free number from 499 downwards.
+doubt, take the next free number from 499 downwards. There is a <uri
+link="https://gitweb.gentoo.org/data/api.git/tree/bin/used_free_uidgids.sh">
+helper script</uri> available in the <uri
+link="https://gitweb.gentoo.org/data/api.git">data/api.git</uri> to aid finding
+the next free UID or GID. Run it by issuing <c>./bin/used_free_uidgids.sh</c>
+inside api.git repo. 
```

Comment:
Given that there's a link to data/api.git immediately above, I think the first `<uri>` tag isn't strictly necessary. The second one isn't very useful either, because (IIUC) the script won't work outside the repo and downloading it from here doesn't make much sense.

Also, I think that the wording should be either "in the data/api.git repository" or "in data/api.git".

Reply:
Done, originally I thought about **bolding** or **otherwise styling** the data/api.git, but plaintext probably works fine too. As you said it's linked above already.